Documentation/contributing/development/images.rst

```diff
@@ -46,6 +46,72 @@ Anyone can build official release images using the make target below.
 
     DOCKER_IMAGE_TAG=v1.4.0 make docker-images-all
 
+Experimental Docker BuildKit and Buildx support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Docker BuildKit allows build artifact caching between builds and
+generally results in faster builds for the developer. Support can be
+enabled by:
+
+::
+
+    export DOCKER_BUILDKIT=1
+
+Multi-arch image build support for arm64 (aka aarch64) and amd64 (aka
```

Comment:
The instructions above this line seem like a generally good idea for all developers. The instructions from this line and below seem to be more specially targeted towards developers interested in buildx and cross-compilation. Maybe we should just separate this line & below into a dedicated section?

```suggestion

Experimental cross-platform builds using Docker Buildx
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Multi-arch image build support for arm64 (aka aarch64) and amd64 (aka
```

Reply:
arm64 support in Cilium images is tracked in the table that is part of this section. I'm not objecting, but IMO it's valuable to keep these together.